src/net.cpp

```diff
@@ -1069,6 +1069,29 @@ void CConnman::AcceptConnection(const ListenSocket& hListenSocket) {
     RandAddEvent((uint32_t)id);
 }
 
+bool CConnman::AddConnection(const std::string& address)
+{
+    int nOutboundFullRelay = 0;
+
+    {
+        LOCK(cs_vNodes);
+        for (const CNode* pnode : vNodes) {
+            if (pnode->m_conn_type == ConnectionType::OUTBOUND) {
```

Comment:
Note for the future, I think we could refactor out this code snippet in some accounting type helper. We're computing `nOutboundFullRelay`/`nOutboundBlockRelay` in diverse parts of the codebase, at least after #19858 

Like returning a struct with an aliased int for each type we have.
```
struct SetConnectionType {
       int  full_relay;
       int   blocksonly;
       ...
};

GetConnectionTypeCount()
```

Reply:
sounds good for a future PR 👍🏽

going to resolve this conversation to keep the review comments focused on these changes